internal/pkg/coordinator/monitor.go

```diff
@@ -202,11 +202,9 @@ func (m *monitorT) ensureLeadership(ctx context.Context) error {
 		}
 		leaders, err = dl.SearchPolicyLeaders(ctx, m.bulker, ids, dl.WithIndexName(m.leadersIndex))
 		if err != nil {
-			if errors.Is(err, es.ErrIndexNotFound) {
-				m.log.Debug().Str("index", m.leadersIndex).Msg(es.ErrIndexNotFound.Error())
-				return nil
+			if !errors.Is(err, es.ErrIndexNotFound) {
+				return err
```

Comment:
don't want to log anymore if the index is missing?

Reply:
No because this monitor creates a document, that will create the index. So no need to log it.